### ISSUE TITLE
Returns `BETA` instead of `ETA`

### ISSUE THREAD

**Issue by FariborzDaneshvar-NOAA** [oceanmodeling/StormEvents#93]
`StormEvent()` function with name=`ETA` and year=2020, returns the same output as name=`BETA` and year=2020. 
For both cases, `storm.name=BETA` and the `storm.track()` is for BETA, so clearly the code is returning BETA instead of ETA.

**Comment by SorooshMani-NOAA** [comment]
@FariborzDaneshvar-NOAA please test. I'm re uploading the image again, it's on `/work2/noaa/nos-surge/smani/sandbox/ondemand-storm-workflow/singularity/imgs/`

**Comment by FariborzDaneshvar-NOAA** [comment]
Thanks @SorooshMani-NOAA I tested `stormevent()` and it worked fine for ETA!

**Comment by FariborzDaneshvar-NOAA** [comment]
@SorooshMani-NOAA with the new fix, `stormevent().name` is returning the right name, but looks like it still returning the old (duplicate) storm track! 
Here are BEST track, 34kt and 50kt isobars I'm getting for both (which both are for BETA, see the second plots)
| storm track lines for BETA 2020 | storm track lines for ETA 2020 | 
| --- | --- |
| ![image](https://github.com/oceanmodeling/StormEvents/assets/132295102/f0c86ca8-8cc3-4dc0-83ca-79c3efd56c80) | ![image](https://github.com/oceanmodeling/StormEvents/assets/132295102/90603036-5bec-42ce-903a-d49aba8df037) |

Here are the actual storm tracks, showing that what I got from `stormevent()` was for BETA:
| BETA 2020 storm track | ETA 2020 storm track |
| --- | --- |
| https://en.wikipedia.org/wiki/Tropical_Storm_Beta_(2020)#/media/File:Beta_2020_path.png | https://en.wikipedia.org/wiki/Hurricane_Eta#/media/File:Eta_2020_path.png |

To summarize, `stormevent(name='ETA', year=2020).name` returns `ETA`, but `stormevent(name='ETA', year=2020).track().name` returns `BETA` and the track lines and dates are corresponding to `BETA`

**Comment by FariborzDaneshvar-NOAA** [comment]
But the `ondemand-storm-workflow` is working fine! Here are perturbed tracks for 24-hr leadtimes of BETA and ETA! 

| BETA 2020 | ETA 2020 |
| --- | --- |
| ![storm_tracks](https://github.com/oceanmodeling/StormEvents/assets/132295102/23eab4d6-2dc2-482a-8e23-7c3a3352c83d) | ![storm_tracks](https://github.com/oceanmodeling/StormEvents/assets/132295102/4f56fea1-0230-4398-bab3-faadff5cc40f) |

**Comment by SorooshMani-NOAA** [comment]
@FariborzDaneshvar-NOAA can you please test `storm_name_2` branch https://github.com/oceanmodeling/StormEvents/tree/bugfix/storm_name_2 and let me know if it works correctly or not?

If the workflow is working we can leave it at previous version of stormevents. but this new branch should fix the issue in the storm events anyway!

**Comment by FariborzDaneshvar-NOAA** [comment]
@SorooshMani-NOAA I tested this new branch and it worked fine! Here is the storm track and COOPS stations for ETA with this new branch. 
![ETA_2020_COOPS_and_USGS_stations](https://github.com/oceanmodeling/StormEvents/assets/132295102/e6a4c0f9-464c-4934-8d17-fa8689ec5965)
Thanks for fixing this. You can close the issue.
